### ISSUE TITLE
It takes a while for the first message to show up on the UI/Screen

### ISSUE THREAD

**Issue by paterik** [vikmeup/SCLAlertView-Swift#206]
I've noticed a strange behaviour on (any) initial show-calls ... takes a while for the first message to show up on the UI/Screen, all further showCalls will be executed perfectly fast-and-fine. I've tested your library in version 0.5.9 with an iPhone 6s, iOS 9.3.2 (13F69).

**Comment by faileon** [comment]
Yup I have the same issue, first call of show is reaaaally slow.

**Comment by devJoshLopez** [comment]
having the same problem too

**Comment by damtsnkff** [comment]
@paterik do you face the same issue on the simulator ?
To confirm, try launching the app on your iPhone, terminate it, then reopen it (not from xcode). Is this still slow ?

**Comment by paterik** [comment]
Hi @damtsnkff, sorry for the late reply ... yes, i've the same issue during simulating my app - in this particular case the initial show seems even take much longer than on my direct device tests.
